### PR TITLE
Resolves name collisions in featurestore-pandas notebook + elaborates descriptions

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
This PR adds the following updates to the `official/feature_store/sdk-feature-store-pandas.ipynb` notebook: 

- Adds UUID to the feature store's name to avoid name collisions during regression test.
- Elaborates on some sections in the notebook.
- Textual corrections based on the new notebook template.
<br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/feature_store/sdk-feature-store-pandas.ipynb [hunk 1]
```diff
@@ -29,7 +29,7 @@
         "id": "JAPoU8Sm5E6e"
```
Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: <p>This cell has 2 headings? Reduce to one heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: <p>Add heading to cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: <p>Add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: <p>add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: <p>add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: <p>add heading, drop bolding</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: <p>add heading, drop bolding</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Comment (krishr2d2) [reply notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: Updated.

Comment (krishr2d2) [reply notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: added.

Comment (krishr2d2) [reply notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: added.

Comment (krishr2d2) [reply notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: updated.

Comment (krishr2d2) [reply notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: updated.

Comment (krishr2d2) [reply notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: updated.

Comment (krishr2d2) [reply notebooks/official/feature_store/sdk-feature-store-pandas.ipynb:29]: updated.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**